core/src/env/srml/types.rs

```diff
@@ -83,3 +77,5 @@ pub type Moment = u64;
 
 /// The default SRML blocknumber type.
 pub type BlockNumber = u64;
+
+impl_empty_flush_for!(AccountId, Hash);
```

Comment:
I don't know if this is an overkill but okay ... :D

Reply:
Also we might want to add additional constraints to the `EnvTypes` so that they are guaranteed to implement `Flush`. This might become a problem for `node-runtime` though ...

Reply:
I am currently thinking about using `#![feature(optin_builtin_traits)]` which could greatly help here.

Reply:
> Also we might want to add additional constraints to the `EnvTypes` so that they are guaranteed to implement `Flush`. This might become a problem for `node-runtime` though ...

Yeah I though about this too - would certainly catch the compilation errors earlier.

> I don't know if this is an overkill but okay ... :D

Perhaps, I thought since we have done this for all primitive types for the universality